adapters/intertech/intertechtest/exemplary/simple-banner.json

```diff
@@ -0,0 +1,113 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "device": {
+      "ip": "123.123.123.123",
+      "ua": "Mozilla/5.0 (iPad; CPU OS 10_3 like Mac OS X)"
+    },
+    "site": {
+      "page": "mypage"
+    },
+    "imp": [
+      {
+        "id": "test-imp-id",
+        "tagid": "test-tag",
+        "banner": {
+          "format": [
+            { "w": 300, "h": 250 }
+          ]
+        },
+        "ext": {
+          "bidder": {
+            "page_id": 1001,
+            "imp_id": 2002
+          }
+        }
+      }
+    ],
+    "cur": ["USD"]
+  },
+  "httpCalls": [
```

Comment:
You can verify the headers, specifically the device related ones, are set correctly given that you have a device object here by including the following:
```
"httpCalls": [
    {
      "expectedRequest": {
        "headers": {
          "Content-Type": [
            "application/json;charset=utf-8"
          ],
          "Accept": [
            "application/json"
          ],
          etc...
``